source/adminguide/virtual_machines.rst

```diff
@@ -165,6 +165,15 @@ To create a VM from a template:
    functions such as mounting additional volumes and ISO images,
    live migration, and graceful shutdown.
 
+   **VMware**
+
+   If the rootDiskController and dataDiskController are not specified for an instance using instance details and
+   these are set to use osdefault in the template or the global configuration, than CloudStack tries to find the
```

Comment:
```suggestion
   these are set to use osdefault in the template or the global configuration, then CloudStack tries to find the
```

Reply:
Are you sure @shwstppr ? I am not a native speaker but I think "if - than" and "when - then" are the proper combinations

Reply:
I feel then is okay here, https://www.google.com/search?channel=fs&q=then+vs+than

Reply:
https://www.google.com/search?channel=fs&client=ubuntu&q=then+vs+than